ExtCore/Collections.List.fs

```diff
@@ -777,3 +777,25 @@ let distinct (list : 'T list) : 'T list =
     // Return the result list.
     result
 
+/// <summary>Returns a new list created by keeping only those elements that are not in the second list.  In the result of 'List.difference list1 list2', the first occurrence of each element of 'list2' in turn (if any) has been removed from 'list1'.</summary>
```

Comment:
as you don’t add docs for params and return value, it would be cleaner to omit <summary> and use just /// one liner.